js/content/common.js

```diff
@@ -196,16 +196,45 @@ let ExtensionLayer = (function() {
 
     // NOTE: use cautiously!
     // Run script in the context of the current tab
-    self.runInPageContext = function(fun){
-        let script  = document.createElement('script');
-        script.textContent = '(' + fun + ')();';
+    self.runInPageContext = function(fun) {
+        let script  = document.createElement("script");
+        script.textContent = '(' + fun + ")();";
         document.documentElement.appendChild(script);
         script.parentNode.removeChild(script);
     };
 
     return self;
 })();
 
+class Messenger {
+    static sendMessage(msgID, info) {
+        window.postMessage({
+            type: "es_" + msgID,
+            information: info
+        }, '*');
+    }
+
+    static addMessageListener(msgID, fn, autoRemove) {
```

Comment:
The option on `addEventListener()` to only fire once is called `once`, should we be using the same name here?

Reply:
Sure